lib/order.rb

```diff
@@ -0,0 +1,21 @@
+class Order
+
+  attr :basket
+
```

Comment:
attr is now depracated - you should use attr_accessor instead. In this instance I would actually use a reader instead of an accessor - again, you want to ensure that any additions or subtractions to the order are controlled.